### DISCUSSION TITLE
如何找到在github領取的作業在哪？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#298] [General]
在AI人工智慧導論點選eeclass公告的作業連結領取作業，建立自己的github作業儲存庫後，若離開當時開啟的github儲存庫網頁，要怎麼再次找到剛建立的作業儲存庫呢？我嘗試的結果就是只要進到AI人工智慧導論課程的github組織(organization)，就可以看到建立的github作業儲存庫。驗證方式是打開作業儲存庫，觀察網址列的網址，會發現與點選eeclass公告的作業連結產生的網址相同，因此是連到同一個儲存庫。
# AI人工智慧導論的組織
![image](https://user-images.githubusercontent.com/86505454/161881784-23a088d6-8c89-4293-8c14-edadae65a80e.png)
## 往下拉，`1102-midterm-report-enoch-lightning`就是我的期中報告作業
![image](https://user-images.githubusercontent.com/86505454/161881811-8151b940-0cc6-4017-9099-5b24dc53e717.png)
## 領取期中報告作業的eeclass作業領取公告
![image](https://user-images.githubusercontent.com/86505454/161882225-45cd68f8-3b2c-40d6-88f8-ceeb31c6d5b3.png)
## `hw1-enoch-lightning`就是我的第一次作業
![image](https://user-images.githubusercontent.com/86505454/161881811-8151b940-0cc6-4017-9099-5b24dc53e717.png)
## 領取第一次作業的eeclass公告
![image](https://user-images.githubusercontent.com/86505454/161882644-a73e11d0-ce1c-4a7a-8124-d685aca1d862.png)
## 再往下拉一點可以看到`git-github-fundamentals-enoch-lightning`，就是我的第0次作業
![image](https://user-images.githubusercontent.com/86505454/161882733-aefe2867-6786-4e76-b906-d5db582e84bc.png)
## 領取第0次作業的eeclass公告
![image](https://user-images.githubusercontent.com/86505454/161883078-3d37d4a5-eafc-43c3-a2fd-9b7f15a9e1c1.png)

**Comment by chia-kai-chang** [comment]
謝謝 @enoch-lightning  的詳細解說
